src/main/java/seedu/address/logic/commands/RedoCommand.java

```diff
@@ -18,6 +18,14 @@
     public static final String COMMAND_WORD = "redo";
     public static final String REDO_COMMAND_USAGE = COMMAND_WORD;
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Redo the recent command. "
+            + "\nCommands that can be redone:"
```

Comment:
I am not sure about this, but is it neccessary to list the commands, that could be redone? Since redo only exist when undo is executed right?

Reply:
I'm pretty sure some troller gonna ask, might as well put it in, since this only shows when you type help redo right.

Reply:
Okay then